qiskit/pulse/commands/sample_pulse.py

```diff
@@ -104,27 +105,34 @@ def draw(self, dt: float = 1,
              style: Optional['PulseStyle'] = None,
              filename: Optional[str] = None,
              interp_method: Optional[Callable] = None,
-             scaling: float = 1, interactive: bool = False):
+             scale_factor: float = 1, interactive: bool = False,
+             scaling: float = 1):
         """Plot the interpolated envelope of pulse.
 
         Args:
             dt: Time interval of samples.
             style: A style sheet to configure plot appearance
             filename: Name required to save pulse image
             interp_method: A function for interpolation
-            scaling: Relative visual scaling of waveform amplitudes
+            scale_factor: Relative visual scaling of waveform amplitudes
             interactive: When set true show the circuit in a new window
                 (this depends on the matplotlib backend being used supporting this)
+            scaling: Deprecated, see `scale_factor`
 
         Returns:
             matplotlib.figure: A matplotlib figure object of the pulse envelope
         """
         # pylint: disable=invalid-name, cyclic-import
+        if scaling:
+            warnings.warn(
+                'The parameter "scaling" is being replaced by "scale_factor"'
+                '', DeprecationWarning, 3)
```

Comment:
There's no need to continue the string to this line. Please remove the `''` and move the comma to the previous line for all instances of this warning

Reply:
Noted, thanks